src/FOM/UserBundle/Resources/translations/messages.de.yaml

```diff
@@ -67,7 +67,7 @@ fom:
         edit_user: 'Benutzer bearbeiten'
         contact_user: 'Kontaktieren Sie den Benutzer per E-Mail'
         edit_group: 'Gruppe bearbeiten'
-        delete_user: 'Benutzer löschen'
+        delete_user: 'Möchten Sie den Benutzer %user_username% wirklich löschen'
```

Comment:
Please add quotes around the template parameter, so it later renders as `Möchten Sie den Benutzer "the user" wirklich löschen`. That way it is visually easier to see it is a parameter.